gm_dotnet_managed/GmodNET/GameConsoleWriter.cs

```diff
@@ -0,0 +1,141 @@
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace GmodNET
+{
+    /// <summary>
+    /// Redirects Console's output to game's console
+    /// </summary>
+    public class GameConsoleWriter : TextWriter
```

Comment:
Please change access modifier for this class from public to internal

Reply:
resolved by https://github.com/GmodNET/GmodDotNet/pull/70/commits/716e96570c55205a362b51c01450dc838b3c420c